recipes/parallel-preprocessor/meta.yaml

```diff
@@ -0,0 +1,80 @@
+# manually update or generated from template
+package:
+    name: parallel-preprocessor
+    version: '0.3'
+source:
+    # calc check sum by command sha256sum
+    sha256: 0f472735203b16e092686003e50e25d2e6dbdeef43da8f14177d6bcb4db86634
+    url: https://github.com/ukaea/parallel-preprocessor/releases/download/v0.3/parallel-preprocessor.zip
+
+build:
+    # number should be generated, or manually update, YYMM pattern here, must be 0 for the first PR
+    number: '0'
+    string: oct_2020
```

Comment:
Is this needed?
```suggestion
```

Reply:
just for me, to know when it is build.  as  0.3 will be there for 1 year, although I may have some update . maybe I should tag as 0.3.1